apps/site/lib/site_web/controllers/schedule/finder_api.ex

```diff
@@ -136,6 +136,13 @@ defmodule SiteWeb.ScheduleController.FinderApi do
         is_current: is_current
       )
 
+    route_ids =
+      if route_id == "Green" do
+        ["Green-B", "Green-C", "Green-D", "Green-E"]
```

Comment:
This is accessible as `GreenLine.branch_ids()` so we don't have to hard-code it.

Also, could probably inline this:

```elixir
route_ids = if(route_id == "Green", do: GreenLine.branch_ids(), else: [route_id])
```